apple/OmnivoreKit/Sources/Views/Article/WebAppView.swift

```diff
@@ -28,7 +49,7 @@ public let readerViewNavBarHeight = 50.0
     }
 
     func makeUIView(context: Context) -> WKWebView {
-      let webView = WebView(frame: CGRect.zero)
+      let webView = WebViewManager.create()
```

Comment:
@jacksonh Should this be calling `.shared` instead?

Reply:
@satindar I left shared in there because I'm using it in testing, but I wanted to have it create a new one every time for now, just to be the same as our previous behavior. 

Once we start using shared we need to remove the old content on dismiss or it will show briefly.